public/typography.css

```diff
@@ -137,6 +137,10 @@ p {
   color: var(--color-foreground-5);
   font-style: italic;
 }
+.txt-error {
```

Comment:
We're just using this in a single place, maybe we can just inline it there?

Reply:
Yeah I wanted to keep the line of doing it, and eventually with more error handling, we could start using something like this. But for now we can do it inline

Reply:
634126e